src/Farmer/Types.fs

```diff
@@ -199,6 +199,10 @@ type LinkedResource =
         | Unmanaged resId -> resId
     member this.Name = this.ResourceId.Name
 
+type CertConfig =
+    | AppServiceCertificate of string
+    | NoCertificate of string
```

Comment:
Might be based to rename this `AppServiceDomain` ro something like that as it's not just about the cert.
`NoCertificate` option seems to be used as a NoDomain option so perhaps we should call it that?

Reply:
Sounds good, will change 👍